src/javascripts/components/hub/airportComponent.js

```diff
@@ -0,0 +1,52 @@
+const buildAirport = (airport) => {
+  let domString = '';
+  domString += `
+        <div class="fancy-card airport">
+        <div class="additional">
+          <div class="user-card">
+            <img width="450" src="${airport.imgUrl}" alt="image of ${airport.airportName}">
+          </div>
+          <div class="more-info shading" >
+            <h1 class="mb-3 mt-3 ml-3 mr-3" style="font-size:1.2rem">${airport.airportName}</h1>
+            <div class="coords">
+              <span class="float-left">Airport Code: ${airport.airportCode}</span>
+              <span>${airport.city} ${airport.state}</span>
+            </div>
+            <div class="coords">
+              <span class="float-left" >International Flights: ${airport.isInternational}</span>
+              <span>${airport.country}</span>
+            </div>
+            <div class="stats">
+              <div>
+                <div class="title">Flights</div>
```

Comment:
i like this section a LOT but think we should make a few changes because of it
since you 
1. are using icons lets kill the titles - icons are self explanatory
2. It feels strange for this stuff to be hard coded - lets add 4 sections to your seed data:
* numFlights
* numRestaurants
* numShuttles
* numHotels
3. Once the data is added modify your domstring so it reads in the correct numbers

Reply:
done